README.md

```diff
@@ -253,11 +243,39 @@ If you have those, follow the steps below:
 4. Run `./build.sh` to generate all chain inits.
 5. Add `./chains/<origin_chain>/<chain_id>/bootnodes` and add the boot nodes (see other chains for examples).
 
+## Signing of transaction
+Mosaic command supports two different kind of signing mechanisms.
+
+1. For geth client mosaic commands support [clef](https://github.com/ethereum/go-ethereum/tree/master/cmd/clef) signer which can be used to sign transactions. Refer official documentation to setup [clef](https://github.com/ethereum/go-ethereum/tree/master/cmd/clef) signer. Clef signer can be configured while starting a mosaic chain with option `--clef-signer` which accepts RPC or IPC endpoint of clef.
+
+    Example 
+    ```bash
+      mosaic start 1405 --origin goerli --clef-signer http://host.docker.internal:8550
```

Comment:
also I didn't see code or instructions to ease user setup clef. Shall we be adding a pointer for it too ?